src/DependencyInjection/ContentApiExtension.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\DependencyInjection;
+
+use Libero\PingController\PingController;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use function array_keys;
+
+final class ContentApiExtension extends Extension
```

Comment:
some docblocks about responsibilities of the classes (e.g. loads `services.xml` into somewhere) may be helpful for the non initiated

Reply:
Doing something like https://matthiasnoback.nl/2013/10/symfony2-some-things-i-dont-like-about-bundles/ might help?

Reply:
Haven't experiences the detailed issues first-hand, but :+1: for explicit references over magic